integrations/upgrade/index.test.ts

```diff
@@ -1706,16 +1836,19 @@ test(
       }
 
       --- ./src/components.css ---
-      @import './typography.css';
+      @import './typography.css' layer(components);
 
+      --- ./src/components.utilities.css ---
       @utility foo {
         color: red;
       }
```

Comment:
I think that technically since the only non `@utility` is an `@import` _with_ a `layer(…)` that this split shouldn't be required. But that feels like an optimization for a separate PR (if we even need that.).